portfolio/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-  <application>YOUR_PROJECT_ID_HERE</application>
+  <application>kaoki-sps-spring20git</application>
```

Comment:
Is “git” supposed to be part of the ID?

I thought they were all username-sps-spring20

Reply:
i updates when i noticed that